projects/gen-ai.md

```diff
@@ -0,0 +1,143 @@
+# Generative AI Observability
+
+>[!IMPORTANT] This project extends LLM Semantic Conventions project
+
+## Description
+
+Generative AI observability is evolving fast. The industry is still learning how to use GenAI products which creates new opportunities and challenges in the observability space. GenAI observability is focused on development time and verbose telemetry. Non-deterministic nature of GenAI creates the need to measure the quality and safety of responses which become part of observability.
+
+There are multiple GenAI-focused observability projects out there such as [Arize Phoenix](https://docs.arize.com/phoenix), [Langfuse](https://langfuse.com/), [LangSmith](https://www.langchain.com/langsmith), [Langtrace](https://langtrace.ai/), [OpenLit](https://github.com/openlit/openlit), [Traceloop](https://www.traceloop.com/docs/introduction) and many others.
+
+Some of these projects are based on OpenTelemetry, others are "inspired by" it, but don't have out-of-the-box integration with OTel.
+
+The scope of GenAI observability includes instrumentations of various GenAI clients, frameworks, response evaluations, as well as model (server-side) instrumentations.
+
+Example Use Cases:
+
+- **Debugging**: Standardizing log data and error reporting for efficient error resolution in LLM applications.
+- **Evaluation**: Collecting signal for evaluation of LLM performance, including response accuracy, context retention, and user feedback.
+- **Cost Tracking**: Monitoring operational costs like computation time and API calls for resource optimization.
+- **AI Safety**: Monitoring for ethical LLM use, including privacy compliance, bias detection, and secure data handling.
+
+For Spans: Instrumentation and debugging guidance for multi-step LLM applications, with recommendations on user privacy and intellectual property security, marking sensitive attributes and events as opt-in.
+For Metrics: Tracking model performance by collecting and correlating data like model names, versions, parameters, latency, and evaluation scores.
```

Comment:
As I've seen them discussed in semconv. Is it worth calling out events/log records here as well? Or is it out of the scope for now?

Reply:
yeah, I rewrote this section - it was there in the original LLM semconv project proposal and I no longer understand what it means :)